src/LayoutManager/ActiveSelectionLayoutManager.ts

```diff
@@ -1,27 +1,57 @@
-import { LayoutManager, buildStandardEvents } from './LayoutManager';
+import { LayoutManager } from './LayoutManager';
 import type { RegistrationContext, StrictLayoutContext } from './types';
 import type { FabricObject } from '../shapes/Object/FabricObject';
 
 /**
- * The ActiveSelectionLayoutManager is exactly as the LayoutManager
- * But what it does differently is that it subscribes the activeSelection
- * modification events to modify the layout of the original group of the
- * children. This is done by swapping target and object in the subscribeTargets
- * method. All the rest is identical
+ * Today the LayoutManager class also takes care of subscribing event handlers
+ * to update the group layout when the group is interactive and a transform is applied
+ * to a child object.
+ * The ActiveSelection is never interactive, but it could contain obejcts from
+ * groups that are.
+ * The standard LayoutManager would subscribe the children of the activeSelection to
+ * perform layout changes to the activeselection itself, what we need instead is that
+ * the transformation applied to the active selection will trigger changes to the
+ * original group of the children ( the one referenced under the parent property )
+ * This subclass of the LayoutManager has a single duty to fill the gap of this difference.
  */
 export class ActiveSelectionLayoutManager extends LayoutManager {
   /**
-   * subscribe to object layout triggers
+   * Subscribe an object to transforms events that will trigger a layout change in the group
+   * This is important only for interactive groups.
+   * @param object
+   * @param context
    */
-  protected attachHandlers(
-    childObject: FabricObject,
+  protected subscribe(
+    object: FabricObject,
     context: RegistrationContext & Partial<StrictLayoutContext>
-  ): (() => void)[] {
-    const { parent, group: activeSelection } = childObject;
+  ) {
+    const { parent, group: activeSelection } = object;
     if (!parent || !activeSelection || activeSelection === parent) {
```

Comment:
in fact you could skip re-subscribining

Reply:
i could but this is not we do in the other group.
This surface another issue anyway.
if i later remove a single object of many in the same parent are remove from the active selection, this will break the group layout.
Meaning that we need to keep multiple events and invalidate more than once.
Not fixing now, at least this works most of the time.

Reply:
Added a test to certify the bug exists